smac/facade/roar_facade.py

```diff
@@ -0,0 +1,130 @@
+import logging
+
+import numpy as np
+
+from smac.tae.execute_ta_run import ExecuteTARun
+from smac.tae.execute_ta_run import StatusType
+from smac.stats.stats import Stats
+from smac.scenario.scenario import Scenario
+from smac.runhistory.runhistory import RunHistory
+from smac.runhistory.runhistory2epm import AbstractRunHistory2EPM, RunHistory2EPM4LogCost, RunHistory2EPM4Cost
+from smac.initial_design.initial_design import InitialDesign
+from smac.intensification.intensification import Intensifier
+from smac.smbo.acquisition import AbstractAcquisitionFunction
+from smac.epm.random_epm import RandomEpm
+from smac.facade.smac_facade import SMAC
+
+__author__ = "Marius Lindauer"
+__copyright__ = "Copyright 2016, ML4AAD"
+__license__ = "3-clause BSD"
+
+
+class ROAR(SMAC):
+
+    def __init__(self,
+                 scenario: Scenario,
+                 tae_runner: ExecuteTARun=None,
+                 runhistory: RunHistory=None,
+                 intensifier: Intensifier=None,
+                 acquisition_function: AbstractAcquisitionFunction=None,
```

Comment:
I don't think an acquisition function is necessary because we don't use a model.